src/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>hello world</title>
```

Comment:
```suggestion
    <title>Hello world</title>
```

It's better to write titles beginning with a capital letters. It looks nicer